eng/pipelines/common/clone-checkout-bundle-step.yml

```diff
@@ -0,0 +1,13 @@
+steps:
```

Comment:
Should Libraries and CoreCLR jobs be updated to use this template? Should just be a couple of files: https://github.com/dotnet/runtime/search?q=Checkout_bundle&unscoped_q=Checkout_bundle

Reply:
Yeah, I initially waved off on it to avoid adding unnecessary changes while everyone's working on the yaml in various ways, but I'll give it a shot.